var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -175,6 +181,11 @@ def _configure_args(self):
                 '--enable-curand',
                 '--enable-cub'
```

Comment:
Sorry I forgot to ask earlier: if the below code is supposed to replace `--enable-cub`, shouldn't we only add that if `spec.satisfies('@:2.20.99')` (i.e. for all versions before `2.21.0`). For example this could be removed and we could add to the below if statement:

```
if '@2.21.0:' in self.spec:
    ...
else:
    configure_args.append('--enable-cub')
```

Reply:
That is right, so I modified accordingly.
Further, to show the benefit of switching from "Package" to "AutotoolsPackage", I used the with_or_without and enable_or_disable condensed forms. Of course switching to cmake one day is another matter.